src/pages/offerings.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE HTML>
+<html lang="en">
+    <head>
+        <title>DevilSec Offers</title>
+        <base target="_blank">
+    </head>
+    <body>
+        <div class="container-fluid row text-center">
+		<h1> What does DevilSec offer? </h1> 
+		<div class="col card text-white bg-secondary mb-3">
```

Comment:
The columns not having any sizing messes up the display of the footer because the page doesnt properly take their size.
Use sized columns like you did in your `connect.html`.